pkg/telepresence/v1/Makefile

```diff
@@ -0,0 +1,31 @@
+# TELEPRESENCE_NAME is the name to use when creating intercepts.
+ifndef TELEPRESENCE_NAME
+$(error "TELEPRESENCE_NAME must be defined in the project's Makefile.")
+endif
+
+# TELEPRESENCE_PORT is the port to use when creating intercepts.
+ifndef TELEPRESENCE_PORT
+$(error "TELEPRESENCE_PORT must be defined in the project's Makefile.")
+endif
+
+# TELEPRESENCE_RUN_CMD is the command to use to start a local server for a
+# Telepresence intercept.
+TELEPRESENCE_RUN_CMD ?=
+
+# TELEPRESENCE_RUN_REQ is a space separated list of prerequisites needed to run
+# a local server for a Telepresence intercept.
+TELEPRESENCE_RUN_REQ +=
+
+################################################################################
+
+.PHONY: intercept
+intercept:
+	PORT=$(TELEPRESENCE_PORT) TELEPRESENCE_PORT=$(TELEPRESENCE_PORT) telepresence intercept $(TELEPRESENCE_NAME) --port $(TELEPRESENCE_PORT) -- $(shell echo $$SHELL)
```

Comment:
As it stands we couldn't really use this for Go projects as BetSecure. I wont link it here, but take a look at the `intercept:` recipe in on of those repos.

TL;DR we have some things to put in the recipe before starting the intercept. It could probably be handled with the addition of a `TELEPRESENCE_INTERCEPT_REQ` variable, but perhaps not very ergonomic.